test/framework/cluster.go

```diff
@@ -899,7 +898,25 @@ func (e *ClusterE2ETest) InstallCuratedPackagesController() {
 	}
 }
 
-func (e *ClusterE2ETest) InstallCuratedPackage(packageName, packagePrefix string, opts ...string) {
+func (e *ClusterE2ETest) SetPackageBundleActive() {
```

Comment:
> As mentioned before, this shouldn't be necessary.

...I might be wrong on this. A change I could have sworn was merged... isn't there.

Reply:
It shouldn't be, but I was hitting errors without it so we either need to fix it on the controller or keep this in.

Reply:
Did you rebase to catch teh package controller changes? If so that might indicate a bug in the packagecontrollerclient if it was able to get through without an active package.

Reply:
I'm not sure if it's a sort issue of speed since it would basically do everything really fast on an e2e test and it would try to install a package before the reconciliation loop finished?